functions.php

```diff
@@ -1319,3 +1340,287 @@ function wp_autoupdates_themes_bulk_actions_handle( $redirect_to, $doaction, $it
 	}
 }
 add_action( 'handle_network_bulk_actions-themes-network', 'wp_autoupdates_themes_bulk_actions_handle', 10, 3 );
+
+/**
+ * Disable auto updates via Ajax.
+ */
+function wp_autoupdates_disable_auto_updates() {
+	$nonce = filter_input( INPUT_POST, 'nonce' );
```

Comment:
I'd highly recommend changing this to `$_POST['nonce']`, core doesn't use the `filter_input` pattern. 

It is also unclear if we can use the filter API at all since it can be optionally disabled and we don't require the filter API anywhere.

Reply:
Okie dokie. I will change that. Thanks Timothy.

Reply:
Fixed with https://github.com/WordPress/wp-autoupdates/pull/61/commits/9b352f1bce847f90730feffe5f5b5ce44ac6de7d